Gordon360/Controllers/AccountsController.cs

```diff
@@ -118,6 +121,9 @@ public async Task<ActionResult<IEnumerable<BasicInfoViewModel>>> SearchWithSpace
             string? minor,
             string? hall,
             string? classType,
+            string? preferredClassYear,
+            string? initialYear,
+            string? finalYear,
```

Comment:
If these are years, should they be `int`s instead of `string`s?

Reply:
Thanks! Just changed this